fcrepo/fcrepo
# Alter preferTags to use enum (Minimal container triples)

Setup new ITs.

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3334

# What does this Pull Request do?
Makes an new enum to allow for identifying include, exclude or silent prefer headers.

Made new `PrefersLdpIT` to hold prefer header tests. Moved several tests from `FedoraLdpIT`  to `PrefersLdpIT`

# How should this be tested?

As this is essentially about minimal container triples.
Make a resource, add some Rdf, and a contained resource.
GET the resource.
See user triples, server managed triples and containment triples.
GET the resource with `Prefer: return=representation; include="https://www.w3.org/ns/ldp#PreferMinimalContainer"`
See user triples and server managed triples.
GET the resource with `Prefer: return=representation; omit="https://www.w3.org/ns/ldp#PreferMinimalContainer"`
See containment triples.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers
